core/src/test/java/fr/sncf/osrd/api/StandaloneSimulationTest.java

```diff
@@ -289,6 +290,45 @@ public void withMarecoAllowanceRanges() throws Exception {
         assertNull(simResult.ecoSimulations.get(0));
     }
 
+    @Test
+    public void withScheduledPoints() throws Exception {
```

Comment:
Ideally, this test would have caught the above bug with stop times; would you mind changing it so that it includes a long stop midway?

Reply:
I've adapted the test to include stops in the path.